pages.fr/common/adb-disconnect.md

```diff
@@ -0,0 +1,7 @@
+# adb disconnect
+
+> Cette commande a été déplacée vers `adb connect`.
+
+- Voir la documentation de la commande originale :
```

Comment:
```suggestion
- Affiche la documentation de la commande originale :
```

Reply:
It's an error.

> Command and example descriptions on pages in French must use the third person singular present indicative tense.

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules

And since you can't conjugate to `voit`, you'll need ton find another verb.

Reply:
Good catch, I'll PR to change this doc, so we can have a discussion with the other maintainers. For now, don't PR on other pages to change "voir" until we decided something.